### ISSUE TITLE
Implement account play time limit. SMSG_PLAY_TIME_WARNING

### ISSUE THREAD

**Issue by heyitsbench** [azerothcore/azerothcore-wotlk#16525]
### Current Behaviour

There is no support for the blizzlike account play time limits.

### Expected Blizzlike Behaviour

On Chinese servers, there was a [limit imposed on the consecutive play time of accounts](https://www.gamespot.com/articles/china-govt-steps-up-limits-on-online-gaming/1100-6131845/). If you played your account for more than three hours, your money and experience rates would be cut in half. After five hours of play, the character would not be able to loot, gain experience, or complete quests. Being logged out for five hours would rest these states and make these rates go back to normal.

### Source

[Various articles](https://www.wowhead.com/news/three-hour-playtime-penalty-for-chinese-players-showing-up-on-the-shadowlands-318590), [in-game strings](https://wow.tools/dbc/?dbc=globalstrings&build=7.3.0.24473#page=1&search=playtime_) (yes it's a Legion build, but the strings exist in 3.3.5 somewhere I promise), as well as [a VMaNGOS commit](https://github.com/vmangos/core/commit/d015f29f1f7828fb9efdc5e7a78e410006c225cd)

### Steps to reproduce the problem

1. Play a character for more than five hours.
2. Observe you are not roleplaying as a Chinese realm player.

### Extra Notes

Do note that the times for the online time restrictions (three and five hours) are baked into the strings, so while server-side the times could be different and changeable, they would not change to the client and would confuse players. I recommend that that value stay hardcoded if this gets implemented. The time to reset the states is *not* baked in, so that could probably be customizable.

Needless to say, this limit should be implemented as an optional config.

### AC rev. hash/commit

c676a828376877e63da77fc1adc81ac2aa3daab0

### Operating system

Windows 10 x64

### Custom changes or Modules

[mod-ptr-template](https://github.com/heyitsbench/mod-ptr-template)

**Comment by heyitsbench** [comment]
There is also nothing in place for parental controls, but I'm still trying to find out how that actually connects to the client if at all.

**Comment by blinkysc** [comment]
Interesting. I see these in us client too it seems

**Comment by Kitzunu** [comment]
Perhaps it could be used for parental control in the US client?

**Comment by heyitsbench** [comment]
If you mean the strings, I'm pretty confident those were translated for all clients, doesn't necessarily mean they were ever used for other regions.

**Comment by blinkysc** [comment]
Yeah don't think used back then, but I think the functionality exists

**Comment by blinkysc** [comment]
<img width="820" height="332" alt="Image" src="https://github.com/user-attachments/assets/c4249d2a-8ad3-4ae9-acbb-5a91ab9dde66" />

**Comment by moostigre** [comment]
was this implemented with #24576 ?
